pxr/usd/usd/plugInfo.json

```diff
@@ -148,7 +148,10 @@
                 "Validators": {
                     "CompositionErrorTest": {
                         "doc": "Validator aims at providing all composition errors, which were generated while composing the stage."
-                    }, 
+                    },
+                    "StageMetadataChecker": {
+                        "doc": "Stages that can be consumed as referencable assets should have a valid 'defaultPrim' declared."
```

Comment:
Its a error and not a warning, so maybe "must have a valid 'defaultPrim' specified."?

Reply:
Updated